CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file based on the
 
 ### Breaking changes
 
+* Rename `network.total.bytes` to `network.bytes` and `network.total.packets`
```

Comment:
I'm worried about this breaking change. 

@andrewkroh How much will this affect packetbeat / auditbeat?
@webmat Did you check how much this effects Metricbeat / Filebeat?

Reply:
It wasn’t previously used by Auditbeat or Packetbeat AFAIK. I have 
https://github.com/elastic/beats/pull/9121 to use the new network.bytes in packetbeat.